examples/leadingones.py

```diff
@@ -71,7 +71,6 @@ def leading_ones(cfg, seed):
 try:
     incumbent = smac.optimize()
 finally:
-    smac.stats.print_stats()
```

Comment:
The optimize function of SMAC looks like

```
        try:
            incumbent = self.solver.run()
        finally:
            self.solver.stats.print_stats()
            self.logger.info("Final Incumbent: %s" % (self.solver.incumbent))
```

So, this would duplicate the output of the stats.